code/datums/uplink_items/uplink_general.dm

```diff
@@ -503,6 +503,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/clothing/glasses/chameleon/thermal
 	cost = 15
 
+/datum/uplink_item/stealthy_tools/thermal
```

Comment:
```suggestion
/datum/uplink_item/stealthy_tools/thermal/chamnightvision
```

Your new datum had the exact same path as regular thermals. I don't think these new glasses are thermals ^v^

Reply:
Good to resolve, outdated. @HMBGERDO